### ISSUE TITLE
Is twitter api broken ?

### ISSUE THREAD

**Issue by harikt** [auraphp/bin#5]
It looks the tweets of releases are not send from twitter of @auraphp account ( https://twitter.com/auraphp/with_replies ) . 

Did it got broken ?

**Comment by pmjones** [comment]
No, I think there's something wrong with the SSL certs on my laptop. I need to look into it.
